### ISSUE TITLE
The bot doesn't close all parenthesis types

### ISSUE THREAD

**Issue by makiki99** [HugoNikanor/reddit-parenthesis-bot#2]
This bot closes only `(`, it doesn't close `{` or `[`.

For example, see this [reddit link](https://www.reddit.com/r/ProgrammerHumor/comments/4j9bpd/found_on_the_dominos_website/d352mxr?context=3).

**Comment by HugoNikanor** [comment]
Well, it is the _parenthesis_ bot, not the _bracket_ bot.

Also,
`(`, `[`, `{`, `⁅`, `⁽`, `₍`, `〈`, `❨`, `❪`, `❬`, `❮`, `❰`, `❲`, `❴`, `⟦`, `⟨`, `⟪`, `⟬`, `⟮`, `⦃`, `⦅`, `⦇`, `⦉`, `⦋`, `⦍`, `⦏`, `⦑`, `⦓`, `⦕`, `⦗`, `⸨`,

**Comment by dmarcuse** [comment]
I think you forgot < >

**Comment by o11c** [comment]
For reference, http://www.unicode.org/Public/8.0.0/ucd/BidiBrackets.txt

**Comment by julianwachholz** [comment]
Yes, this is crucial for use of the site and needs to be supported.
